examples/records/records.bal

```diff
@@ -89,4 +99,33 @@ public function main() {
     // record, else `()` is returned.
     int? english = grades["english"];
     io:println(english);
+
+    // A mapping constructor expression used when creating a record value
+    // can also include a spread field, referring to another mapping value.
+    // When a spread field is specified, all the fields of the relevant
+    // mapping value are added to the new record value being created.
+    // A spread field is used with `address` to include the individual address
+    // entries in `address` when creating `anne`.
+    Student anne = {
+        name: "Anne",
+        age: 18,
+        grades: {
+            maths: 70,
+            physics: 80,
+            chemistry: 55
+        },
+        ...address
+    };
+    io:println(anne);
+
+    // Using a mapping constructor expression with `var` (i.e., no contextually expected
+    // type) results in a mapping value where the inferred type is a record type based
```

Comment:
```suggestion
    // type) results in a mapping value, where the inferred type is a record type based
```
We can have a comma before the "where" conjunction here to increase clarity and understandability since the sentence is long.